lib/app/item.ex

```diff
@@ -168,6 +168,32 @@ defmodule App.Item do
     end)
   end
 
+  @doc """
+  `person_with_item_and_timer_count/0` returns a list of number of timers and items per person.
+  Used mainly for metric-tracking purposes.
+
+  ## Examples
+
+  iex> person_with_item_and_timer_count()
+  [
+    %{name: nil, num_items: 3, num_timers: 8, person_id: 0}
+    %{name: username, num_items: 1, num_timers: 3, person_id: 1}
+  ]
+  """
+  def person_with_item_and_timer_count() do
+    sql = """
+    select p.person_id, p.name, COUNT(distinct i.id) as "num_items", count(distinct t.id) as "num_timers"
```

Comment:
If we're writing raw  `SQL`, prefer the keywords e.g. `SELECT`, `AS`, `FROM` to be `ALLCAPS` for readability.

e.g: 
https://github.com/dwyl/mvp/blob/faba54760ab0435653434aec7c9f801096fbbd8b/lib/app/item.ex#L151-L154

Reply:
Simplified: https://github.com/dwyl/mvp/issues/245#issuecomment-1377238534